content/fr/container-orchestration.md

```diff
@@ -4,14 +4,14 @@ status: Completed
 category: Concept
 ---
 
-L'orchestration de [Conteneurs](/fr/container/) fait référence à la gestion et à l'automatisation du cycle de vie d'applications conteneurisées au sein d'environnements dynamiques.
-Réalisée à l'aide d'un orchestrateur de conteneurs (la plupart du temps, [Kubernetes](/fr/kubernetes)), elle permet les déploiements, le passage à l'échelle (automatique), (l'auto-)remédiation et le monitoring.
+L'orchestration de [Conteneurs](/fr/container/) fait référence à la gestion et à l'automatisation du cycle de vie d'applications [conteneurisées](/containerization/) au sein d'environnements dynamiques.
```

Comment:
Missing "/fr" prefix in the containerization URL.

Reply:
Fixed. Thank you for catching that! (We have per-locale relative-URLs setup for the OTel website, therefore we don't need such prefixed — and so I forgot it here.)